### PR TITLE
34427 map slice content build

### PR DESCRIPTION
**Pull request by GcioNapoleon**
## Description

This is the content build commit for this ticket.

https://github.com/department-of-veterans-affairs/va.gov-team/issues/34427

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/includes/image_and_static_map_dynamic.liquid [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  {% endcapture %}
+  {% include src/site/components/image.drupal.liquid with
+      class = "facility-img"
+      image = fieldMedia.entity.image
+      onclick = onclick
+  %}
+  <div data-widget-type="facility-map-satellite-locations"
+       data-facility="{{ facilityId }}">
+  </div>
+</div>
```
Comment (srsuddath) [inline src/site/includes/image_and_static_map_dynamic.liquid:14]: it looks like it wants a newline here

File: src/site/includes/image_and_static_map_satellite_main.liquid [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  {% endcapture %}
+  {% include src/site/components/image.drupal.liquid with
+      class = "facility-img"
+      image = fieldMedia.entity.image
+      onclick = onclick
+  %}
+  <div data-widget-type="facility-map-satellite-main-office"
+       data-facility="{{ facilityId }}">
+  </div>
+</div>
```
Comment (srsuddath) [inline src/site/includes/image_and_static_map_satellite_main.liquid:14]: and here

File: src/site/includes/vet_centers/address_phone_image_map.liquid [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    %}
+    {% include "src/site/includes/image_and_static_map.liquid" with
+        facilityMedia = vetCenter.fieldMedia
+        facilityId = "vc_0626V"
+    %}
+  </section>
+
+</div>
+
+{% endcomment %}
```
Comment (srsuddath) [inline src/site/includes/vet_centers/address_phone_image_map.liquid:84]: we should not be committing this much commented out code

Comment (srsuddath) [reply src/site/includes/vet_centers/address_phone_image_map.liquid:84]: we should remove this code block?

**Review by srsuddath — CHANGES_REQUESTED** [review]

File: src/site/includes/image_and_static_map_satellite_main.liquid (hunk 2 above)
Comment (GcioNapoleon) [reply src/site/includes/image_and_static_map_satellite_main.liquid:15]: done

File: src/site/includes/image_and_static_map_dynamic.liquid (hunk 1 above)
Comment (GcioNapoleon) [reply src/site/includes/image_and_static_map_dynamic.liquid:15]: done

**Review by srsuddath — APPROVED** [review]

**Review by mdewey — APPROVED** [review]
LGTM

**Pull request merged**